app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] One of the project requirement is that, users should be able to confirm their emails to be fully registered. I have noticed that the user's model is missing `:confirmable` attribute which makes it possible to confirm user emails.